src/index.js

```diff
@@ -0,0 +1,253 @@
+  projectLi.addEventListener('mouseenter', function showTrash() { this.lastElementChild.lastElementChild.classList.add('d-inline'); });
+  projectLi.addEventListener('mouseleave', function hideTrash() { this.lastElementChild.lastElementChild.classList.remove('d-inline'); });
+
+  projectLi.addEventListener('click', function renderProjectArea() {
+    projectId = this.getAttribute('data');
+    renderProjects(mainProjects.projectsArr);
+    renderTask(
+      mainProjects.projectsArr[projectId].tasksArr,
+      mainProjects.projectsArr[projectId].taskName,
+    );
+    localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+  });
+  projectLi.lastElementChild.addEventListener('click', function runderAfterRemove() {
+    mainProjects.removeProject(this.parentElement.getAttribute('data'));
+    if (mainProjects.projectsArr[0]) {
+      renderProjects(mainProjects.projectsArr);
+      renderTask(
+        mainProjects.projectsArr[projectId].tasksArr,
+        mainProjects.projectsArr[projectId].taskName,
+      );
+    }
+    localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+
+    projectId = 0;
+  });
+  return projectLi;
+}
+
+function renderProjects(arr) {
+  let projectsList = document.querySelector('.projects-list');
+  projectsList.innerHTML = '';
+  if (arr.length === 0) {
+    document.querySelector('.card-container').classList.add('d-none');
+  } else {
+    document.querySelector('.card-container').classList.remove('d-none');
+  }
+  let count = 0;
+  arr.forEach(elem => {
+    const projectLi = createProjectElement(elem, count);
+    projectsList = document.querySelector('.projects-list').appendChild(projectLi);
+    count += 1;
+  });
+}
+
+function addEventListenerToTasksForm() {
+  document.querySelector('#addTaskId').addEventListener('submit', function findSubmitOfTask(e) {
+    e.preventDefault();
+    mainProjects.projectsArr[projectId].addTask(this.children[0].value);
+    this.children[0].value = '';
+    renderTask(
+      mainProjects.projectsArr[projectId].tasksArr,
+      mainProjects.projectsArr[projectId].taskName,
+    );
+    renderProjects(mainProjects.projectsArr);
+    localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+
+    document.querySelector('.task-items-cont').lastChild.classList.add('animate-in');
+  });
+}
+
+function addEventListenerToProjectsForm() {
+  document.querySelector('#addProjectId').addEventListener('submit', function findSubmitOfProject(e) {
+    e.preventDefault();
+    mainProjects.addProject(this.children[0].value);
+    this.children[0].value = '';
+    renderProjects(mainProjects.projectsArr);
+    localStorage.setItem('project', JSON.stringify(mainProjects.projectsArr));
+  });
+}
+
+// testing 123
```

Comment:
- [ ] This is not a commented code but it's not an informative comment. We should also delete this.